beef/template.json

```diff
@@ -1,14 +1,38 @@
 {
-  "title":"",
-  "description":"",
-  "date":"",
-  "url":"",
-  "prepTime":"",
-  "cookTime":"",
-  "totalTime":"",
-  "servings":,
-  "directions":[],
-  "notes":"",
-  "ingredients":[],
-  "tags":[]
+  "title":"Smoked Peppered Beef Jerk",
+  "description":"Spiced up, irresistible beef jerky. This jerky is doused in a beer, soy sauce, and black pepper marinade, and smoked low and slow for an epic protein-packed snack.",
+  "date":"2019-10-1",
+  "url":"https://www.traegergrills.com/recipes/beef/peppered-beef-jerky",
+  "imageUrl":"https://www.traegergrills.com/images/en-us/Shared/images/recipes/banners/Beef/Peppered-Beef-Jerky_RE_HE.jpg",
```

Comment:
I appreciate the link to the image. However, I think it would be good to go ahead and download the image, and store it in the repo. Then whenever we get a front end made, we can create pretty recipe cards with the image.

Store it here:  beef/images/smoked-peppered-beef-jerk.jpg